Assets/Code/src/Runtime/Scripts/PhysicsJoints.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
I already removed the PhysicsJoints directory as I've already integrated it into the Player Authoring script. We don't need these anymore. Please delete this entire directory and make sure that the changes affect actual players in-game.

Like I said, please refrain from developing in your own sandbox and make sure it works and integrates with the rest of the system.